tests/testthat/test-class-workbook.R

```diff
@@ -76,6 +69,17 @@ test_that("wb_set_col_widths", {
 
 })
 
+test_that("set_col_widths errors when inconsistent lengths are supplied", {
```

Comment:
Maybe `"set_col_widths informs when inconsistent lengths are supplied"`?
But that's just nitpicking